boards/stm32f412gdiscovery/src/main.rs

```diff
@@ -139,8 +142,7 @@ unsafe fn set_pin_primary_functions() {
     });
     PinId::PA03.get_pin().as_ref().map(|pin| {
         pin.set_mode(Mode::AlternateFunctionMode);
-        // AF7 is USART2_RX
-        pin.set_alternate_function(AlternateFunction::AF7);
+        // AF7 is USART2_RX pin.set_alternate_function(AlternateFunction::AF7);
```

Comment:
Why is this commented out?

Reply:
It seems that I've messed up this line after solving some merge conflicts. There was no reason for it to be commented. Thanks for spotting it!